From b82b1a4881704cd38f1c172a232c80da6b083347 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 27 Jun 2024 11:33:00 -0700
Subject: [PATCH] ==== docs improvements for DAMON ====

Subject: Docs/damon: minor fixups and improvements

As the title says.  Fixup typos, clarify regions merging operation with
recent change, add access pattern snapshot example use case, and improve
readability of the design document by reorganizing/wordsmithing some
sections and adding links to usage documents for easy browsing.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/n9QvkCan | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/n9QvkCan

diff --git a/damon_meta_changes/n9QvkCan b/damon_meta_changes/n9QvkCan
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

